src/connectors/idagio.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const symphonySelector = '.player-PlayerInfo__infoEl--2jhHY span:nth-child(3) span';
+const movementSelector = '.player-PlayerInfo__infoEl--2jhHY span span span';
+const directorSelector = '.player-PlayerInfo__recordingInfo--15VMv>span:first-child span';
+const recordingInfoSelector = '.player-PlayerInfo__recordingInfo--15VMv';
+const pauseButtonSelector = '.player-PlayerControls__btn--1r-vy:nth-child(2) .util-IconLabel__component--3Uitr span';
+
+Connector.playerSelector = '.player-PlayerBar__bar--2yos_';
+
+Connector.artistSelector = '.player-PlayerInfo__infoEl--2jhHY span:first-child';
+
+Connector.getTrack = () => getCurrentTrack();
+
+Connector.getAlbum = () => getCurrentSymphony();
```

Comment:
I would not create new functions, but this way also looks fine for me. But I think it's redundant to wrap function call in an arrow function. `Connector.getTrack = getCurrentTrack` is better.